source/core/parameters.cc

```diff
@@ -1100,6 +1125,12 @@ namespace Parameters
               particles[i].position[2]          = prm.get_double("z");
               particles[i].velocity[2]          = prm.get_double("w");
               particles[i].pressure_location[2] = prm.get_double("pressure z");
+              particles[i].masses               = 4.0/3.0*3.14159265359*particles[i].radius*particles[i].radius*particles[i].radius*prm.get_double("density");
```

Comment:
You should use the PI from deal.II.
https://www.dealii.org/current/doxygen/deal.II/namespacenumbers.html